### PR TITLE
Richard 2020-jan

### PR DESCRIPTION
**Pull request by madmongo1**
First blog entry.

### CODE REVIEW

File: _posts/2020-01-31-RichardsJanuaryUpdate.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+Management takes time.
+
+Something I learned from Vinnie very quickly is that for this reason, interfaces to public objects should provide
+the bare minimum functionality that users can demonstrate a need for. Adding a new public method or class should only
+happen after careful consideration of the consequences.
+
+## Supporting all Toolchains
+
+Another consideration I have never had to encounter before is that Boost.Beast is designed to work on every compiler
+that "robsutly supports" C++11.
```
Comment (vinniefalco) [inline _posts/2020-01-31-RichardsJanuaryUpdate.md]: spelling

File: _posts/2020-01-31-RichardsJanuaryUpdate.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+Boost.Beast is a fantastic piece of work. It's Vinnie's baby, and I am privileged to be asked to hold it in my 
+hands. 
+
+I'm never going to take issue with a mother looking to protect her cubs.
+
+Furthermore, having a legitimate excuse to interact with the other maintainers of Boost on Slack is a pleasure.
+These people are some of the brightest minds on the planet. I live in hope that some of this brilliance will
+rub off.
+
+If you work with C++, I highly recommend that you join the (Cpplang)[slack://open?team=Cpplang] slack channel.
```
Comment (vinniefalco) [inline _posts/2020-01-31-RichardsJanuaryUpdate.md]: You have to direct people to the signup page `slack.cpp.al`

File: _posts/2020-01-31-RichardsJanuaryUpdate.md [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+
+If you'd like to contact me to discuss my experiences I'd be happy to receive a message on slack.
```
Comment (vinniefalco) [inline _posts/2020-01-31-RichardsJanuaryUpdate.md]: Slack is capitalized

**Comment by madmongo1** [comment]
3 issues resolved and force-pushed. Thank you.

**Pull request merged**